docs/rules/no-template-curly-in-string.md

```diff
@@ -0,0 +1,33 @@
+# Warn when using template string syntax in regular strings (no-template-curly-in-string)
+
```

Comment:
"disallow template literal placeholder syntax in regular strings"

(To fit with other rule docs)